protocol/avdl/keybase1/teams.avdl

```diff
@@ -185,6 +185,25 @@ protocol teams {
     TeamShowcase showcase;
   }
 
+  record TeamMemberRole {
+    UID userID;
+    string username;
+    FullName fullName;
+    TeamRole role;
+  }
+
+  record TeamInfo {
```

Comment:
This naming may be too generic if it's only used for untrusted team info.

Reply:
I don't feel strongly about it, but being more generic makes it more likely to be reused. I think the fact that the rpc name has untrusted in the name is enough. Happy to change it if you think it's better.